src/utils/line.ts

```diff
@@ -51,54 +58,138 @@ export const formatCreditEvents = (
   price: BigNumber = BigNumber.from(0),
   events: LineEventFragResponse[]
 ): CreditEvent[] => {
-  return events.map((e: any): CreditEvent => {
-    const { __typename, amount, token, credit, timestamp, value = unnullify(0, true) } = e;
+  return events.map((event: any): CreditEvent => {
+    const { id, timestamp, __typename, amount, value, position } = event;
     return {
-      id: credit?.id,
+      id,
       __typename,
       timestamp,
       amount,
+      // value: unnullify(value, true),
       value,
-      token: token?.id,
-      currentValue: price.mul(value),
+      token: position.token.id,
+      // currentValue: price.mul(value),
     };
   });
 };
 
+// TODO: Modify/adjust this to replace mergeCollateralEvents (formatCollateralEvents)
+// given that there are 3 duplicative uses of collateral events in this file!!
+export const formatEscrowToCollateralEvents = (escrow: AggregatedEscrow | undefined): CollateralEvent[] => {
```

Comment:
deposits arent events. Events are a separates would be `escrow.events` not `escrow.deposits`

Reply:
yep, I know - will remove this function entirely.

Reply:
removed